content/blog/posts/devops-daycamp-2015.rst

```diff
@@ -0,0 +1,39 @@
+DevOps DayCamp 2015
+===================
+:date: 2015-10-29
+:author: Amanda Kelner
+:slug: devops-daycamp
```

Comment:
This slug exists already in `content/blog/posts/devops-daycamp.rst`. One of them needs to be changed. This one could be `devops-daycamp-2015`